test/views/email/PasscodeErrorViewSpec.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.email
+
+import views.html.email.PasscodeErrorView
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import play.twirl.api.Html
+import views.ViewBaseSpec
+
+class PasscodeErrorViewSpec extends ViewBaseSpec {
+
+  val injectedView: PasscodeErrorView = inject[PasscodeErrorView]
+  lazy val view: Html = injectedView()(user, messages, mockConfig)
+
+  lazy implicit val document: Document = Jsoup.parse(view.body)
+
+  "The passcode error view" should {
+
+
+    "have the correct document title" in {
+      document.title shouldBe "You need to start again - Business tax account - GOV.UK"
+    }
+
+    "have the correct heading" in {
+      document.getElementsByClass("heading-large").text() shouldBe "You need to start again"
```

Comment:
You could rewrite this as `elementText(".heading-large")` to keep it consistent with the other tests that use CSS selectors below.